src/components/Dropdown/Dropdown.tsx

```diff
@@ -101,22 +111,31 @@ export function Dropdown(props: PropsWithChildren<DropdownProps>): JSX.Element {
     }
 
     if (e.key === 'ArrowDown') {
-      updateFocusedItem(focusedIndex + 1);
+      if (alwaysSelectOption && focusedIndex === items.length-1) {
+        updateFocusedItem(0);
+      } else {
+        updateFocusedItem(focusedIndex+1);
```

Comment:
nit: spacing here and a few more places in this file

Reply:
I made an item to add https://typescript-eslint.io/rules/space-infix-ops to our eslint

Reply:
the spacing rule is just no spaces between operators right? or is there other spacing i also need to fix

Reply:
we want the spacing between operators so like `focusedIndex + 1` here instead of `focusedIndex+1`
I'm definitely ok ignoring adjusting these for now since this is an autofixable eslint rule.